### PR TITLE
W 11673756 manager processboard se

### PR DESCRIPTION
**Pull request by sebermuley**
New topic: Manage Process Project Tasks
+ (Added missing topics: Deactivate/disable 2FA)
+ Editing

Dear SME,
please review for correctness and completeness of purpose and content.
Thank you.

### CODE REVIEW

**Review by roelofmeyer — DISMISSED** [review]

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc [hunk 1]
```diff
@@ -1,91 +1,94 @@
...
+RPA Manager returns to the *<category name>* view. Your new project is sorted into the view according to your workspace settings.
 
-If you already have evaluated a process with a third-party tool, you can copy and edit a similar automation project directly inside a category.
-
-The copy contains all information from the original project including the most recent revision of all workflows with the exception of the bot assignments to test and productive configurations, and the history of the workflows.
+== Copy an Automation project
 
-To complete copying of the project, first transfer the project to the *Build* phase. Then open it in the MuleSoft RPA Builder. The project then receives its own identification values. The copied project can only transition to the *Test* phase after it has been uploaded from the RPA Builder to the RPA Manager.
+A copy of an automation project contains all information from the original project including the most recent revision of all workflows with the exception of the bot assignments to test and productive configurations.
```
[the remark this answers is not in the document]
Comment (dhirsch1) [reply rpa-manager/modules/ROOT/pages/processautomation-manage.adoc:59]: I am not sure what this means: "to test and productive configurations."

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc [hunk 2]
```diff
@@ -110,37 +115,131 @@ Remove automation projects you no longer need. This cannot be undone and is neit
...
+. Open the *Project Management* view of the *Process Automation* module.
+. Click on the name of the category of the project.
+. In the *<category name>* view, click the *Remove* icon image:delete-icon.png[trash symbol,1.5%,1.5%] on the panel of the project you want to remove.
 . Confirm the prompt.
 
+== Manage Process Project Tasks
+
+You can manage all tasks that arise in association with automating a process via an agile process board if you have <<projectdata-useprocessboard, enabled it for the project>>.
+
+A task consists of a name, a description, an owner, a priority, a due date, and an effort estimate. You can drag tasks across the board to reflect their progress.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-manage.adoc:129]:
```suggestion
A task consists of a name, a description, owner, priority, due date, and effort estimate. You can drag tasks across the board to reflect their progress.
```

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc [hunk 3]
```diff
@@ -110,37 +115,131 @@ Remove automation projects you no longer need. This cannot be undone and is neit
...
+
+* *To-do*
++
+The *To-do* column contains all the tasks that have not been processed yet. You create new tasks here. If you have <<projectdata-phasereleasetasks, enabled the creation of automatically generated tasks for phase transitions>>, RPA Manager creates a new corresponding task during each phase transition. Remember to move tasks that are being processed into the *In Progress* area.
+* *In Progress*
++
+The *In Progress* column contains all the tasks that someone is currently working on. Remember to move tasks that have been completed into the *Completed* area.
+* *Completed*
++
+The *Completed* column contains all the tasks that have already been completed. You can edit and move completed tasks, too. If you have enabled the creation of automatically generated tasks for the phase transitions, the button for the transition to the next phase disabled until this task has been moved to the *Completed* column.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-manage.adoc:141]:
```suggestion
The *Completed* column contains all tasks that have already been completed. You can edit and move completed tasks, too. If you have enabled the creation of automatically generated tasks for the phase transitions, the button for the transition to the next phase disabled until this task has been moved to the *Completed* column.
```

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc [hunk 4]
```diff
@@ -110,37 +115,131 @@ Remove automation projects you no longer need. This cannot be undone and is neit
...
+. Confirm the deletion.
+
+=== Move a Task
+
+Move tasks within a column to sort them or across columns to reflect their progress:
+
+. <<openprocessboard, Open a process board>>.
+. Drag a task item to its new place.
+
+If you have enabled the creation of automatically generated tasks for the phase transitions, move this task to the *Completed* column after all other tasks have been completed to enable the transition of the project to the next phase.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-manage.adoc:211]:
```suggestion
If you enabled the creation of automatically generated tasks for the phase transitions, move this task to the *Completed* column after all other tasks have been completed to enable the transition of the project to the next phase.
```

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc [hunk 5]
```diff
@@ -153,11 +154,36 @@ Both the computer on which the RPA Manager is installed, and the mobile device o
 . In the *Profile* view, click *Enable Two-Factor Authentication*.
 . Follow the instructions in the window *Enable Two-Factor Authentication*.
 . Click *Enable Two-Factor Authentication*.
-. Log in using your password and 2FA.
+
+Log in using your password and 2FA.
+
+== Disable Two-Factor Authentication for Your User
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:160]: Maybe this should be "Disable Two-Factor Authentication for Yourself"

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc [hunk 6]
```diff
@@ -1,91 +1,94 @@
...
-If you already have evaluated a process with a third-party tool, you can copy and edit a similar automation project directly inside a category.
-
-The copy contains all information from the original project including the most recent revision of all workflows with the exception of the bot assignments to test and productive configurations, and the history of the workflows.
+== Copy an Automation project
 
-To complete copying of the project, first transfer the project to the *Build* phase. Then open it in the MuleSoft RPA Builder. The project then receives its own identification values. The copied project can only transition to the *Test* phase after it has been uploaded from the RPA Builder to the RPA Manager.
+A copy of an automation project contains all information from the original project including the most recent revision of all workflows with the exception of the bot assignments to test and productive configurations.
 
-Copy an existing similar project to save time:
+Copy a project to automate a similar process faster:
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-manage.adoc:61]:
```suggestion
Copy a project to automate a similar more quickly:
```

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc [hunk 7]
```diff
@@ -1,91 +1,94 @@
...
+. Click on the name of the category in which you want to create the new project.
+. In the *<category name>* view, click the *Copy* icon image:copy-icon.png[copy symbol,1.5%,1.5%] on the panel of the project you want to copy.
+. Check and edit the form *Copy process*:
 +
-include::rpa-manager::partial$form-automationproject.adoc[]
+For an explanation of the properties, see <<form-createprocess, *Create process*>>.
 . Click *Save*.
-. Open the project in xref:rpa-builder::index.adoc[MuleSoft RPA Builder] and upload it to RPA Manager.
+
+To complete copying of the project, first transfer the project to the *Build* phase. Then open it in the xref:rpa-builder::index.adoc[RPA Builder]. The project then receives its own identification values. The copied project can only transition to the *Test* phase after it has been uploaded from the RPA Builder to the RPA Manager.
```
[the remark this answers is not in the document]
Comment (dhirsch1) [reply rpa-manager/modules/ROOT/pages/processautomation-manage.adoc:71]: Consider using an ordered list here.

To complete copying the project:

1. Transfer the project to the *Build* phase. 
2. Open it in the xref:rpa-builder::index.adoc[RPA Builder]. 
+
The project receives its own identification values.

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc [hunk 8]
```diff
@@ -110,37 +115,131 @@ Remove automation projects you no longer need. This cannot be undone and is neit
...
+
 == Analyze the Financial Aspects of a Process
 
-Open the xref:processoperations-financeanalysis.adoc[view *Finance Analysis*] to record, monitor, and analyze the financial aspects of a process:
+Open the xref:processoperations-financeanalysis.adoc[*Finance Analysis* view] to record, monitor, and analyze the financial aspects of a process:
 
-. In a category in the *Process Management* view of the *Process Automation* module, click the *Finance Analysis* icon image:financeanalysis-icon.png[ symbol,1.5%,1.5%] the panel of the project you want to analyze.
+. Open the *Project Management* view of the *Process Automation* module.
+. Click on the name of the category of the project.
+. In the *<category name>* view, click the *Finance Analysis* icon image:financeanalysis-icon.png[ symbol,1.5%,1.5%] the panel of the project you want to analyze.
```
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/processautomation-manage.adoc:219]: It looks like this is missing an "of" or "for" before "the panel of the project you want to analyze."

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc (hunk 5 above)
Comment (dhirsch1) [inline rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:160]: For consistency with later text: Disable Two-Factor Authentication for Yourself

**Review by dhirsch1 — APPROVED** [review]

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc (hunk 6 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/processautomation-manage.adoc:61]: @dhirsch1 there is a difference between process and project ( see above). Therefor I included the word 'process': '
Copy a project to automate a similar process more quickly:'

File: rpa-manager/modules/ROOT/pages/processautomation-manage.adoc (hunk 8 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/processautomation-manage.adoc]: ... on ...

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**